.travis.yml

```diff
@@ -0,0 +1,50 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.4
+      env:
+        - EXECUTE_CS_CHECK=true
```

Comment:
CS check requires "our" `.php_cs` file. You may copy it from `ppi/framework` or just remove the CS_CHECK requirement from `travis.yml`.